datadog_cluster_agent/manifest.json

```
@@ -0,0 +1,34 @@
+{
+  "display_name": "Datadog Cluster Agent",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "datadog_cluster_agent",
+  "metric_prefix": "datadog.cluster_agent.",
+  "metric_to_check": "",
```

```suggestion
  "metric_to_check": "datadog.cluster_agent.admission_webhooks.certificate_expiry",
```
Does this integration have a tile? Otherwise, this will allow the dashboard to show up